Procfile

```diff
@@ -1,3 +1,3 @@
 postdeploy: ./scripts/postdeploy.sh
-worker: php bin/console messenger:consume async_priority_high async
+worker: php bin/console messenger:consume async_priority_high async --time-limit=3600
```

Comment:
> Some services (like Doctrine's EntityManager) will consume more memory over time. So, instead of allowing your worker to run forever, use a flag like messenger:consume --limit=10 to tell your worker to only handle 10 messages before exiting (then the process manager will create a new process). There are also other options like --memory-limit=128M and --time-limit=3600.

Le worker redémarra toutes les heures ça sera déjà ça pour libérer de la ressource